client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -143,6 +143,9 @@
     @Option(names = "--session", paramLabel = "<session>", description = "Session property (property can be used multiple times; format is key=value; use 'SHOW SESSION' to see available properties)")
     public final List<ClientSessionProperty> sessionProperties = new ArrayList<>();
 
+    @Option(names = "--session-user", paramLabel = "<user>", description = "Impersonating session username")
```

Comment:
I didn't come up with a better description. Suggestions?

Reply:
"Username to impersonate"